packages/itwinui-css/src/expandable-block/expandable-block.scss

```diff
@@ -2,219 +2,195 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @use '../mixins';
 
-@mixin iui-expandable-block {
+// ----------------------------------------------------------------------------
+
+.iui-expandable-block {
   --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon);
   --_iui-expandable-block-expander-icon-transform: none;
-  @include iui-expandable-block-variant;
-  @include iui-expandable-block-size;
+
+  --_iui-expandable-block-border: 1px solid var(--_iui-expandable-block-border-color);
+  --_iui-expandable-block-border-color: var(--iui-color-border);
+  --_iui-expandable-block-background-color: var(--iui-color-background);
+  --_iui-expandable-block-content-padding: var(--iui-size-s);
+  --_iui-expandable-block-header-background-color-disabled: var(--iui-color-background-disabled);
+  --_iui-expandable-block-header-background-color-hover: var(--iui-color-background-hover);
+  --_iui-expandable-block-header-focus-offset: -2px;
+  --_iui-expandable-block-header-focus-thickness: 2px;
+
+  &[data-iui-expanded='true'] {
+    --_iui-expandable-block-header-bottom-border-radius: 0;
+  }
+
+  --_iui-expandable-block-expander-icon-size: var(--iui-size-m);
+  --_iui-expandable-block-header-gap: var(--iui-size-s);
+  --_iui-expandable-block-header-padding: var(--iui-size-s);
 
   display: flex;
   flex-direction: column;
+  isolation: isolate;
   width: 100%;
   border-radius: var(--iui-border-radius-1);
 
   &[data-iui-expanded='true'] {
     --_iui-expandable-block-expander-icon-transform: rotate(90deg);
```

Comment:
this can be combined with the data-iui-expanded rule above (line 21)

Reply:
Right, combined